src/components/sbb-file-selector/readme.md

```diff
@@ -0,0 +1,101 @@
+The `sbb-file-selector` is a component which allows user to select one or more files from storage devices.
+
+It has two different display options based on the value of the `variant` property: by default a `sbb-button` is displayed, 
```

Comment:
Similar question/suggestion as in accordion. Could this be structured into subtitles?

Reply:
Done in [b757080](https://github.com/lyne-design-system/lyne-components/pull/1894/commits/b757080eb29d0c5d03c4a0e5edf5e769d2832fc4)